plugins/markdown/komand_markdown/actions/markdown_to_pdf/action.py

```diff
@@ -10,7 +10,7 @@
 def makePDF(html: str, path: str) -> str:
     infile = path + "str.html"
     outfile = path + "tmp.pdf"
-    with open(infile, "w") as f:
+    with open(infile, "w", encoding="utf-8") as f:
```

Comment:
```suggestion
    with open(infile, "w", encoding="utf-8") as file_:
```

Reply:
Would it be `file_` or `file`? 
Does the underscore mean it is an unused variable?